### DISCUSSION TITLE
Allow Environment variables as command line params to inso cli

### DISCUSSION THREAD

**Discussion by abbasc52** [Kong/insomnia#3699] [Ideas]
**Is your feature request related to a problem? Please describe.**
Environment Variable can hold some sensitive information such as client secret. We cannot check-in such secrets in repo which blocks us from running inso.

**Describe the solution you'd like**
Allow passing environment variables via command-line argument similar to how newman(postman cli) provides: -
`
--env-var "<environment-variable-name>=<environment-variable-value>"
Allows the specification of environment variables via the command line, in a key=value format. Multiple CLI environment variables can be added by using --env-var multiple times, like so: --env-var "foo=bar" --env-var "alpha=beta".
`

**Describe alternatives you've considered**
Finding and replacing environment variables in CI

**Comment by denissignoretto** [comment]
I've a similar problem. Running inso in a CI/CD pipeline I need to pass secrets to Insomnia Env Variables.

**Comment by lukasosterheider** [comment]
Is there any solution to this already?
We have similar issues. Postman/Newman support this functionality.

**Comment by nch3ng** [comment]
Is there any way to update variables through cli? This would be helpful for automation
